web-server/algorand_transaction/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "algorand_transaction"
+version = "0.1.0"
+authors = ["Tsundzukani <trnghondzweni@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+algorand_core = {path = "../algorand_core", version = "0.1.0"}
+ring = "0.16.19"
+rand = "0.8.3"
```

Comment:
We probably don't want to pull in these 2 dependencies (since both depends on std iirc), and rather parameterize the functions that use the functionality from these crates. 

There is a library that just adds that traits implemented by rand that might be no_std compatible. Maybe use that instead.

For ring: I know that there have been work on making a subset of ring no_std compatible, so it might be worth checking for that. Otherwise we should look at rather passing something (like a ring fn) from the caller.